tests/custom_metric/workflow_test.py

```diff
@@ -0,0 +1,35 @@
+import uuid
+
+from temporalio import activity
+from temporalio.testing import WorkflowEnvironment
+from temporalio.worker import Worker
+
+from custom_metric.worker import ExecuteActivityWorkflow
+
+_TASK_QUEUE = "custom-metric-task-queue"
+
+activity_counter = 0
+
+
+async def test_sleep_for_days_workflow():
+
+    @activity.defn(name="print_message")
+    async def print_message_mock():
+        global activity_counter
+        activity_counter += 1
+
+    async with await WorkflowEnvironment.start_time_skipping() as env:
```

Comment:
```suggestion
async def test_sleep_for_days_workflow(client: Client):

    @activity.defn(name="print_message")
    async def print_message_mock():
        global activity_counter
        activity_counter += 1

```

I think we should encourage use of the shared dev server and only encourage use of time skipping server when it's needed (other tests may be unnecessarily using time skipping test server, but they shouldn't IMO)